frontend/isoko/src/pages/BusinessDashboard/BusinessDash.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
+import NavbarComponent from '../components/NavbarComponent';
```

Comment:
Also, I think this should be '../../components/NavBarComponent'
I know this is a merge conflict but I had to change it in order to get it to run.

Reply:
same

Reply:
is this fixed?